src/server/routes.test.js

```diff
@@ -37,6 +37,12 @@ describe("Function: routes", () => {
   afterEach(() => {
     jest.clearAllMocks();
   });
+  
+  it("Should call get", () => {
+    process.env.NODE_ENV = "production"
```

Comment:
Not sure I like this, shouldn't be using production in a test

Reply:
Removed this test again.